src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1069,3 +1074,38 @@ export const disableSmsVerificationsForUser = (
       return transformMongoError(error)
     },
   ).map(() => true)
+
+/**
+ * Checks if the given form field should be updated.
+ * This currently checks if the admin has exceeded their free sms limit.
+ * @param form The form which the specified field belongs to
+ * @param formField The field which we should perform the update for
+ * @returns ok(form) If the field can be updated
+ * @return err(PossibleDatabaseError) if an error occurred while performing the required checks
+ * @return err(SmsLimitExceededError) if the form admin went over the free sm limit
+ * while attempting to toggle verification on for a mobile form field
+ */
+export const shouldUpdateFormField = (
+  form: IPopulatedForm,
+  formField: IField,
+): ResultAsync<
+  IPopulatedForm,
+  PossibleDatabaseError | SmsLimitExceededError
+> => {
+  const formAdminId = String(form.admin._id)
```

Comment:
nit: this can be instantiated after the `if` block below since it's not used until then.

Reply:
sure!